.github/workflows/ci.yaml

```diff
@@ -67,11 +68,11 @@ jobs:
           echo "Deploying serve"
           docker compose up -d --build
 
-      - name: Determine and set Serve URL
+      - name: Set Serve URL
         working-directory: ${{env.working-directory}}
         run: |
           curl --version | head -n 1
-          STUDIO_URL=`awk -F '= ' -v replace="'" '/STUDIO_URL/{gsub(replace, "", $NF); print $NF}' studio/settings.py`
+          STUDIO_URL=http://studio.127.0.0.1.nip.io:8080
```

Comment:
No need to fetch this from the settings file. It is always the same in CI. Plus, it failed when using the new IP property.